### ISSUE TITLE
Enable Resource data Wrapping, turned of by global withoutWrapping

### ISSUE THREAD

**Issue by ChrisSFR** [writingink/wink#132]
Just installed wink to our project as a test. When I create a post and want to publish it, I get the following message:
![bildschirmfoto 2019-01-17 um 12 10 48](https://user-images.githubusercontent.com/5312693/51314900-3b3c6e00-1a51-11e9-8ee5-315e2cc2b027.png)

Seems like the author_id is not set in the save for me. Any ideas?
Currently setting up a new project for testing, to check if my project interferes with wink.

I got the Post Published info box, but no post is saved to my database.

**UPDATE**
In our AppServiceProvider we had
Resource::withoutWrapping();
set.
That influenced the Wink Resources also.

**Comment by ChrisSFR** [comment]
Still investigating.

So the other is set in the wink/edit.vue 
`this.form.author_id = data.author_id || '';`

in the fillForm(data) method, which is triggered in the vue mounted event.

the loadResources method works also: I get the author via /wink/api/teams
`if (!this.form.author_id && this.authors) {
                        this.form.author_id = this.Wink.author.id;
                    }`

and the Wink object is also set in the layout.blade.php

`    window.Wink = {"unsplash_key":null,"path":"wink","author":{"name":"Regina Phalange","avatar":"https:\/\/secure.gravatar.com\/avatar\/edb0e96701c209ab4b50211c856c50c4?s=80","id":"f8eef16b-71f8-4707-9458-21f5f512f9c1"}};`


so I still don't know, why author_id is not set :/

**Comment by ChrisSFR** [comment]
Thanks for your answer.
I'll fork wink and compile it with nom run dev to enable vue dev tools, to inspect and trace the vue properties on Monday.

**Comment by ChrisSFR** [comment]
The authors data attribute is undefined, when I inspect it with vue devtools.
Thereby the post General Setting has no option for the author select
`<select name="author_id" class="input"
                        v-model="form.author_id"
                        id="author_id">
                    <option v-for="author in authors" :value="author.id">{{author.name}}</option>
                </select>`

authors is undefined because the wink/api/team route returns a team collection which is not wrapped in data.
And the response is accessed by response.data.data
` this.http().get('/api/team').then(response => {
                    this.authors = response.data.data;

                    if (!this.form.author_id && this.authors) {
                        this.form.author_id = this.Wink.author.id;
                    }
                });`

This leads to an undefined for me.
Don't know why it is only for me. :/

**Comment by ChrisSFR** [comment]
When I change the response.data.data to response.data It works for me.
But I wonder, why the check if authors is set backdoor don't work for me.
```
if (!this.form.author_id && this.authors) {
                        this.form.author_id = this.Wink.author.id;
                    }
```

**Comment by ChrisSFR** [comment]
Noticed it's only me and my project where Team and TagsResources aren't wrapped in data. Don't know why atm.

**Comment by ChrisSFR** [comment]
OK, got it.
In our AppServiceProvider we had 
`Resource::withoutWrapping();`
set.
That influenced the Wink Resources also.
Any idea how to enable it for Wink?

**Comment by ChrisSFR** [comment]
merged with #190
